integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -151,7 +151,11 @@ master:
   #   timeoutSeconds: 5
   #   failureThreshold: 2
   tolerations: []
-  podAnnotations: {}
+  podAnnotations:
```

Comment:
Is it possible to leverage the `PrometheusMetricsServlet` in the `metrics` section? 
The reason I don't want to add these default pod annotations is that, for people not using the monitor, they don't need to have unnecessary and probably confusing annotations on their pods.

Reply:
Done, I put the annotations in the metrics section, so to use the monitor we must enable the metrics part.